test/Common.cpp

```diff
@@ -57,9 +58,9 @@ boost::filesystem::path testPath()
 	return {};
 }
 
-std::string EVMOneEnvOrDefaultPath()
+std::string GetEnvOrDefaultPath(const char* env_name, const char* lib_name)
```

Comment:
```suggestion
std::string envOrDefaultPath(char const* env_name, char const* lib_name)
```
But actually this should really use strings, even if you pass it on to functions that take C-strings.